src/wp-includes/class-wp-query.php

```diff
@@ -442,6 +442,7 @@ class WP_Query {
 	 * via pre_get_posts hooks.
 	 *
 	 * @since 3.1.1
+	 * @var bool
```

Comment:
Please add an empty comment line between the `@since` tag line and the new `@var` tag line.

Reply:
Push the updated code with new commit.